aspnet/CircularGauge/Legend.md

```diff
@@ -24,10 +24,10 @@ By default, the legend will not be displayed in the circular gauge. You can enab
 
 {% endhighlight %}
 
-![ASPNET CircularGauge Legend image1](Legend_images/Legend_img1.png)
+![ASP.NET CircularGauge Legend image1](Legend_images/Legend_img1.png)
 
 
-[Click](http://asp.syncfusion.com/demos/web/circulargauge/legend.aspx) here to view the online demo sample for  legend in the circular Gauge.
+[Click](https://ej2.syncfusion.com/javascript/demos/#/material3/circular-gauge/legend.html) here to view the online demo sample for  legend in the circular Gauge.
```

Comment:
double space given.
<img width="299" alt="image" src="https://github.com/syncfusion-content/asp.netwebforms-docs/assets/44014283/6962bb46-37ef-4cfa-a55c-d0eb6afa640f">
And Control Name First Char should be in Caps.

Reply:
DONE